crates/contract_history/src/bin/export_contracts.rs

```diff
@@ -0,0 +1,18 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let target_dir = std::path::Path::new("pytest/common_lib/current_contracts");
```

Comment:
does this mean this must be run from the root dir?

Reply:
I am not sure what is the expected workflow, maybe adding that to the readme would help